tests/PHPStan/Analyser/LegacyNodeScopeResolverTest.php

```diff
@@ -8236,11 +8236,11 @@ public function dataCallables(): array
 				'$closure()',
 			],
 			[
-				'Callables\\Bar',
+				'*ERROR*',
 				'$arrayWithStaticMethod()',
 			],
 			[
-				'float',
+				'mixed',
 				'$stringWithStaticMethod()',
```

Comment:
These callables are calling non-static methods. Should I add more test data or is this good enough?

Reply:
Please add similar cases that actually work, something like `$arrayWithStaticMethoForReal()` whose result type will really be `'Callables\\Bar'`.

Reply:
To do so, we'd need to add static methods to the data sample. Previously I was told not to adjust test data. Or is that just for specific `bug-###.php` files?

Reply:
Adding stuff to existing files and adding asserts is fine 😊 Just in case of rule tests, make sure that the lines of already existing errors do not shift (so for example do not add new use statements at the top).

Reply:
@ondrejmirtes perfect! I definitely broke this rule in my last commit, but it also was a bit exploratory as I needed up deep in the weeds. But I think this latest iteration is closer.